templates/vue-starter-template/app/pages/account/address/index.vue

```diff
@@ -0,0 +1,185 @@
+<script setup lang="ts">
+import { ApiClientError } from "@shopware/api-client";
+import type { Schemas } from "#shopware";
+
+const { user, defaultBillingAddressId, defaultShippingAddressId, refreshUser } =
+  useUser();
+const { resolveApiErrors } = useApiErrorsResolver("account_addresses");
+const { pushError } = useNotifications();
+
+const {
+  customerAddresses,
+  loadCustomerAddresses,
+  deleteCustomerAddress,
+  setDefaultCustomerBillingAddress,
+  setDefaultCustomerShippingAddress,
+} = useAddress();
+
+const deletingAddresses = ref<Set<string>>(new Set());
+
+function handleAddAddress() {
+  navigateTo("/account/address/new");
+}
+
+function handleEditAddress(addressId: string) {
+  navigateTo(`/account/address/edit/${addressId}`);
+}
+
+async function handleDeleteAddress(addressId: string) {
+  deletingAddresses.value.add(addressId);
+
+  try {
+    await deleteCustomerAddress(addressId);
+    await loadCustomerAddresses();
+    await refreshUser();
+  } catch (error) {
+    if (error instanceof ApiClientError) {
+      const errors = resolveApiErrors(error.details.errors);
+      for (const error of errors) {
```

Comment:
rename inner `error` from the loop into `errorMessage` to avoid variable shadowing